cartoframes/dataset.py

```diff
@@ -105,6 +107,29 @@ def set_geodataframe(self, gdf):
     def get_table_name(self):
         return self._table_name
 
+    def get_dataset_info(self):
+        if self._is_saved_in_carto:
+            if self._dataset_info is None:
```

Comment:
This means we are caching `dataset_info` so we could have locally an outdated version of the metadata in some cases. If we decide doing it this way we should have a method or `force=True` param to refresh the dataset_info.

What do you think?

Reply:
Really there is something wrong here. I am not taking into account that the synchronization state can change. Let me think about the best way to do it